src/main/java/org/shanerx/tradeshop/utils/ListManager.java

```diff
@@ -228,22 +223,23 @@ private void setGameMatList() {
 			}
 		}
 
-		// Remove all non obtainable materials that we have found
+		// Remove all non-obtainable materials that we have found
 		for (NonObtainableMaterials mat : NonObtainableMaterials.values()) {
 			gameMats.remove(mat.toString());
 		}
 
-		//Adds any strings that have been added the the AddOnMats list to the autocomplete list
+		//Adds any strings that have been added the AddOnMats list to the autocomplete list
 		gameMats.addAll(addOnMats);
 	}
 
 	private void updateDirections() {
 		directions.clear();
-		ArrayList<BlockFace> allowed = new ArrayList<>(Arrays.asList(BlockFace.DOWN, BlockFace.WEST, BlockFace.SOUTH, BlockFace.EAST, BlockFace.NORTH, BlockFace.UP));
 
-		for (String str : Setting.ALLOWED_DIRECTIONS.getStringList()) {
-			if (allowed.contains(BlockFace.valueOf(str)))
-				directions.add(BlockFace.valueOf(str));
+		for (String str : Setting.CHEST_DIRECTIONS.getStringList()) {
+			try {
+				directions.add(RelativeDirection.valueOf(str));
+			} catch (IllegalArgumentException | NullPointerException ignored) {
+			}
```

Comment:
I would actually throw an exception as the admin most likely misconfigured stuff.
If throwing is too dramatic, at the very least log a fat warning to the console.

Reply:
Exception is definitely too dramatic, I'll add a log message for it though.

Reply:
Actually, no this exception is only triggered by blanks or null being read(which shouldn't really happen anyways). this is more likely to be caused by owners improperly formatting then entering a value wrong, so I think a silent fail is better.